contracts/currency_networks_tlbc.md

```diff
@@ -1,6 +1,7 @@
 ---
-layout: contracts
+
 title: Currency Networks on Trustlines Blockchain
+sidebar_label: Currency Networks
 ---
 
 **Commit**: [trustlines-protocol/contracts@baebb30](https://github.com/trustlines-protocol/contracts/commit/baebb308d9ef868408c3449250aa9d963cc61d71)
```

Comment:
@cducrest what do we need this for?

Reply:
I don't know, what info?

Reply:
I think we should not have this file here

Reply:
Leaving it for now there. Will talk with @andfletcher about it.